content/en/scale-agent/concepts/dynamic-account-options.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Migrate Clouddriver Accounts to the Agent
+linkTitle: Dynamic Accounts
+description: >
+  Learn how to migrate account definitions from Clouddriver to the Armory Agent dynamically.
+---
+
+## Dynamic accounts managed by the Armory Agent
+The Agent, and all Agent replicas, can dynamically leverage account data sourced from cluster Clouddriver plugin. Migrated accounts are dynamic and account migration does not result in duplicate database entries or inconsistent state. Dynamic Accounts can also be generated, modified, or deleted from the Agent. The account definitions are dynamic and any account activity on the Agent ise automatically detected and sent to Agent pods, without the impact of downtime.
+
+Dynamic accounts accounts are are supported by the
+{{< linkWithTitle "dynamic-accounts.md" >}} endpoints. The  `zoneId` flag is used to keep each account instance unique. 
+
+> Dynamic Accounts are only available on Agents with network connectivity to the target clusters.
+
+## Configuration details
+This feature is a limited availability beta release and requires the use of custom agent and service plugins (available from your technical account manager).
+
+Dynamic Account configuration parameters are added to the Agent plugin deployment file. The required parameters must be updated as shown here:
+
+``` yaml
+- account:
+  - storage:
+    - enabled: true 
+    - kubernetes: 
+      - enabled: true
+```
+With the the configuration in place you can begin migrating and managing accounts on the Agent. Dynamic accounts can be manually or automatically managed. 
+
+### Manual configuration
+Accounts can be pulled from the Clouddriver to the cluster Agents using the {{< linkWithTitle "dynamic-accounts.md" >}}).
+
+For each account three requests are required to activate a migrated account on the Agent and set that account to `Active`:
+1. First use a `GET /armory/accounts/{accountId}` request to scan for account data. You can also use the `GET /armory/accounts/{accountId}` endpoint to source a specific account.
+2. Next, use the migrate `POST /armory/accounts` request to migrate the accounts to the Agent. This can be done as a batch process. Identified accounts are sent to the agent and set to an `INACTIVE` state.
```

Comment:
This only transfers the account to the kubesvc_accounts table with an INACTIVE state, it is not sent to Agent yet so I wouldn't say migrated to Agent but migrated to kubesvc_accounts, or maybe we can leave the term 'migrate' to avoid confusion.